gno.land/pkg/gnoweb/url/url.go

```diff
@@ -229,28 +229,27 @@ func ParseGnoURL(u *url.URL) (*GnoURL, error) {
 // NoEscapeQuery generates a URL-encoded query string from the given url.Values,
 // without escaping the keys and values. The query parameters are sorted by key.
 func NoEscapeQuery(v url.Values) string {
-	// Encode encodes the values into “URL encoded” form
-	// ("bar=baz&foo=quux") sorted by key.
 	if len(v) == 0 {
 		return ""
 	}
-	var buf strings.Builder
+
 	keys := make([]string, 0, len(v))
 	for k := range v {
 		keys = append(keys, k)
 	}
-	slices.Sort(keys)
+	sort.Strings(keys)
+
+	var pairs []string
 	for _, k := range keys {
 		vs := v[k]
-		keyEscaped := k
-		for _, v := range vs {
-			if buf.Len() > 0 {
-				buf.WriteByte('&')
+		if len(vs) == 0 {
+			pairs = append(pairs, k)
+		} else {
+			for _, val := range vs {
+				pairs = append(pairs, k+"="+val)
 			}
-			buf.WriteString(keyEscaped)
-			buf.WriteByte('=')
-			buf.WriteString(v)
 		}
 	}
-	return buf.String()
+
+	return strings.Join(pairs, "&")
 }
```

Comment:
This method was copied from the Go `url` package to encode the query. I wanted to stick to the original behavior while disabling escaping. The reason is to keep the same semantics as `url.Values.Encode()`. If we want to change this, we have to reimplement the original `Encode` method here, and I'm honestly not sure about this choice.

For the implementation part, you don't have to rewrite the entire method; you can just add:
```go
if len(v) > 0 {
			buf.WriteByte('=')
			buf.WriteString(v)
}
```
so we can stick to the original code.